sycl/include/sycl/info/aspects_deprecated.def

```diff
@@ -2,3 +2,5 @@ __SYCL_ASPECT_DEPRECATED(int64_base_atomics, 7, "use atomic64 instead")
 __SYCL_ASPECT_DEPRECATED(int64_extended_atomics, 8, "use atomic64 instead")
 // Special macro for aspects that don't have own token
 __SYCL_ASPECT_DEPRECATED_ALIAS(usm_system_allocator, usm_system_allocations, "use usm_system_allocations instead")
+__SYCL_ASPECT_DEPRECATED(usm_restricted_shared_allocations, 16, "deprecated in SYCL 2020")
```

Comment:
@gmlueck - It looks like this is in neither SYCL 1.2.1 nor SYCL 2020. Reading the definition of it, it does not seem to have a direct alternative. Is it okay to deprecate it (and remove it in an later release) or do we need an extension to have an alternative?